pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -536,4 +539,28 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	err = InfraSAs.addServiceAccount(
+		InfraGCControllerServiceAccountName,
+		authorizationapi.ClusterRole{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: GCControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// GCController.podStore.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
```

Comment:
I saw some others specifying persmissions per method rather than per resource - which is why I did it this way. If it should be changed, I'm happy to make that change.